scripts/mcp-health-check.sh

```
@@ -0,0 +1,271 @@
+fi
+
+# 6. Test uvx package installation (quick test)
+log_info "Testing uvx package installation capability..."
+
+# Test with a lightweight package
+TEST_PACKAGE="cowsay"
+if uvx --help | grep -q "from" 2>/dev/null; then
+    # Try to list available packages (this doesn't install anything)
+    if command -v timeout >/dev/null 2>&1; then
+        timeout 10s uvx "$TEST_PACKAGE" --help >/dev/null 2>&1
+        local exit_code=$?
```

Shellcheck (0.10.0)

[error] 'local' is only valid in functions.

(SC2168)